linux_os/guide/system/auditing/auditd_configure_rules/file_group_ownership_var_log_audit/ansible/ubuntu.yml

```diff
@@ -0,0 +1,44 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Get audit log files
+  command: grep -iw ^log_file /etc/audit/auditd.conf
+  failed_when: false
+  register: log_file_exists
+
+- name: Parse log file line
+  command: awk -F '=' '/^log_file/ {print $2}' /etc/audit/auditd.conf
```

Comment:
You can avoid this task and use Ansible filters with `log_file_exists` directly on other tasks. Alternatively, you should use `set_fact` instead of setting a register with `command`.